crates/analyzer/src/symbol_path.rs

```diff
@@ -180,10 +180,33 @@ impl From<&syntax_tree::ExpressionIdentifier> for SymbolPathNamespace {
     }
 }
 
+#[derive(Copy, Debug, Clone, PartialEq, Eq)]
+pub enum GenericSymbolPathKind {
+    Identifier,
+    IntegerBased,
+    IntegerBaseLess,
+    IntegerAllBit,
+    RealExponent,
+    RealFixedPoint,
+}
+
+impl GenericSymbolPathKind {
+    pub fn to_kind_name(&self) -> String {
```

Comment:
How about implementing `Display` instead of `to_kind_name`.
Through `Display`, `to_string` is provided and stringify by `display!`, `format!` can be used.

```rust
impl fmt::Display for GenericSymbolPathKind {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        let mut text = String::new();
        ... // construct text
        text.fmt(f)
    }
}
```

Reply:
I implemented `fmt::Display` for `GenericSymbolPathKind `.